spec/mercure.md

```diff
@@ -479,27 +356,43 @@ and in subscription events.
 
 See (#subscription-events).
 
-The `mercure` claim of the JWS **CAN** contain a JSON object under the `payloads` key.
-This JSON document **MUST** have selectors as keys, and user-defined data as values.
+Each entry in the `mercure.subscribe` claim of the JWS **CAN** contain a JSON object under the `payload` key.
 
-The value associated with the first topic selector matching the topic of the subscription
+The value associated with the first topic matcher matching the topic of the subscription
 **MUST** be included under the `payload` key in the JSON object describing a subscription in
 the subscription API and in subscription events.
 
 Example JWT document containing payloads:
 
 ~~~ json
 {
-    "subscribe": ["https://example.com/foo", "https://example.com/bar/baz"]
-    "payloads": {
-        "https://example.com/bar/{val}": {"custom": "data only available for subscriptions matching this selector"},
-        "*": {"data": "available for all other topics"}
-    }
+    "subscribe": [
+        {
+            "match": "https://example.com/foo",
+            "payload": {
+                "custom1": "data only available for subscriptions to this topic"
+            }
+        },
+        {
+            "match": "https://example.com/bar/:id",
+            "matchType": "urlpattern",
```

Comment:
now the case is not important? why not CamelCase like the matchtypes used inside query parameters?

Reply:
fixed